### DISCUSSION TITLE
Weight of iCub(s)

### DISCUSSION THREAD

**Discussion by DanielePucci** [robotology/community#61] [Show and tell]
| iCubGenova01 | iCubGenova02 (v. 2.5) with battery or dummy | iCubGenova03 | iCubDarmstadt01 | iCubGenova04 |
| :-: | :-: | :-: | :-: | :-: |
| 30.3 | 33.1 | 27.5 | 30.4 | 33.33 |

The iCubs were weighted without the skin under the feet's soles. In the case it is installed, add to the above weights `0.2 Kg` (see #56)

For the sake of completeness, we intend as:
iCubGenova01:  head v2, arms v1, SEA legs (without SEA and feet v2)

iCubGenova03:  head v2, arms v2, legs v1 , feet v2, 

iCubGenova04: includes the IMU at the waist and dummy battery.

@traversaro @francesco-romano @maggia80 @fabrizio69 @super-ste @Mick3Lozzo

**Comment by pattacini** [comment]
Finally I know what exact numbers to give when guests will ask for those weights!

Thanks @DanielePucci

**Comment by DanielePucci** [comment]
:dancers:

**Comment by super-ste** [comment]
Please remove version tag (except from iCubGenova02) as they're misleading.
The only way to classify our robot is givin' out the exact description of the way they're made.
e.g. for iCubGenova01
iCub V1 Body 
head V2
SEA legs (without SEA)
:sweat_smile:

**Comment by DanielePucci** [comment]
Ack.

**Comment by alecive** [comment]
@DanielePucci it would be nice if you modify your first post according to @super-ste 's suggestions :+1:

**Comment by DanielePucci** [comment]
Done, checks are welcome.

**Comment by dogoepp** [comment]
Hi Daniele,

What about putting all this weight information in the official 
documentation of iCub ? I think that it might be easier to find there.

**Comment by DanielePucci** [comment]
Hi @dogoepp,
nice idea. Volunteers are welcome.

**Comment by DanielePucci** [comment]
![20140430_094753](https://cloud.githubusercontent.com/assets/5045846/10754979/5c696efe-7c9b-11e5-9dc0-d0880e1c2797.jpg)
![20140430_111627](https://cloud.githubusercontent.com/assets/5045846/10754980/5c6b3784-7c9b-11e5-9a0c-bb41d2005cb8.jpg)

**Comment by traversaro** [comment]
For the weights on `iCubGenova01` and `iCubGenova03` the photos were shot on 30/04/2014, so they reflect the situation of the robots at that point in time.

**Comment by DanielePucci** [comment]
For consistency checks, it would be awesome if our friends in Paris @rlober @mingxing-liu could weight their robots :)

**Comment by francesco-romano** [comment]
Added `iCubDarmstadt01` as in https://github.com/robotology/icub-support/issues/118

**Comment by DanielePucci** [comment]
:100:

**Comment by rlober** [comment]
@DanielePucci 

Next time I am down in the cave I will check our iCubs' body mass indexes!

**Comment by DanielePucci** [comment]
:+1:

**Comment by DanielePucci** [comment]
@spada1978

**Comment by DanielePucci** [comment]
The dummy has been re-designed. Then, the weight is changed accordingly.

**Comment by DanielePucci** [comment]
For further information on the previous dummy weight, see #56

**Comment by traversaro** [comment]
@DanielePucci what is the status of the models commited in the repositories?

**Comment by DanielePucci** [comment]
I sincerely don't remember, but I have flashes of a conversation with you where I asked you to add additional weight on the dummy. We shall check @traversaro @francesco-romano

**Comment by gabrielenava** [comment]
Last week I weighed the green iCub (iCubGenova04, v2.5 with battery pack) without the skin under the feet's soles and this is the result:

![photo_2017-05-31_13-33-51](https://cloud.githubusercontent.com/assets/12396934/26630153/cf8b2230-4605-11e7-99cd-cba4d1b6cff4.jpg)

i.e., the same the weight of the purple (as expected)

**Comment by fjandrad** [comment]
![photo_2018-08-29_10-37-31](https://user-images.githubusercontent.com/11043189/44776099-8ecc1f00-ab77-11e8-8587-20e7a3a79ffa.jpg)

iCubGenova04 on the scale. 
**Total weight = 33.33kg**
Weight of  the current model 33.073kg
@DanielePucci 
Moved here and deleted the other 2 comments.

**Comment by DanielePucci** [comment]
Nice @fjandrad. Can we however update the table on the first https://github.com/robotology/QA/issues/61#issue-110842856?
